identity-wallet/src/state/actions.rs

```diff
@@ -115,7 +115,11 @@ mod bindings {
         UpdateCredentialMetadata { payload: UpdateCredentialMetadata },
         #[serde(rename = "[User Journey] Cancel")]
         CancelUserJourney,
-        #[serde(rename = "[User Data] Query")]
-        UserDataQuery { payload: UserDataQuery },
+        #[serde(rename = "[Search] Query")]
+        SearchQuery { payload: SearchQuery },
+        #[serde(rename = "[Search] Add Recent Search")]
+        AddRecentSearch { payload: AddRecentSearch},
```

Comment:
There's a whitespace missing between `AddRecentSearch` and `}`. Please run `cargo fmt`.